django/thunderstore/utils/tests/test_makemigrations.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from thunderstore.utils.makemigrations import StubStorage, is_makemigrations_check
+
+
+def test_utils_stub_storage():
+    storage = StubStorage()
+    with pytest.raises(NotImplementedError):
+        storage.save("test", b"test")
+
+
+def test_utils_is_makemigrations_check_true(mocker):
```

Comment:
These two test cases could be parametrized :trollface:

Reply:
I did consider 😄